### DISCUSSION TITLE
Entities name are not getting converted to column name in @Filter condition

### DISCUSSION THREAD

**Discussion by shasaxena-equinix** [Adrinalin4ik/Nestjs-Graphql-Tools#16] [Q&A] [answered]
My entity name is in camel case but when queries are generated in where clause it is still taking the camelcase version of the column name hence getting error.

**Comment by Adrinalin4ik** [comment, marked as answer]
It was mentioned in the description that db models should be in snake case. Postgres doesn't like came case.
